src/EnergyPlus/InternalHeatGains.cc

```diff
@@ -341,7 +376,7 @@ namespace InternalHeatGains {
 
                     // Number of people calculation method.
                     {
-                        auto const peopleMethod(AlphaName(4));
+                        auto const peopleMethod(IHGAlphas(4));
```

Comment:
`auto` will create `std::string` and do a copy. At the very least, use `auto const &`. But preferably use `const std::string_view` or `const std::string &` because `auto` is not cool, just annoying.

Reply:
Totally agree.
That was preexisting and all I did was a fancier kind of search and replace. I'm happy to make the change but you mentioned a forthcoming refactor so i don't know if it's worth it?

Reply:
Guess not.